bizdemo/easy_note/cmd/api/main.go

```diff
@@ -49,6 +52,19 @@ func main() {
 			}
 			return jwt.MapClaims{}
 		},
+		LoginResponse: func(ctx context.Context, c *app.RequestContext, code int, token string, expire time.Time) {
```

Comment:
add some Handling Logic to return self-customed code response
- method not found 
- 404 not found
- panic and recovery

Reply:
i got it, thx